pydatalab/src/pydatalab/bokeh_plots.py

```diff
@@ -394,7 +431,70 @@ def selectable_axes_plot(
         )
         plot_columns = [table] + plot_columns
 
-    layout = column(*plot_columns)
+    if plot_points and plot_line:
+        from bokeh.layouts import row
+
+        show_lines_btn = Button(
+            label="✓ Show lines", button_type="primary", width_policy="min", margin=(2, 5, 2, 5)
+        )
+        show_points_btn = Button(
+            label="✓ Show points", button_type="primary", width_policy="min", margin=(2, 5, 2, 5)
+        )
```

Comment:
I think for the sake of ease, we can keep only the button for showing the points or not (i.e., always show the line in the background). Perhaps the line in this case could also be made a little bit thicker.